modules/mod_translation/support/translation_po.erl

```diff
@@ -26,10 +26,11 @@
 
 
 %% @doc Take the list of found labels per module and generate all po files for those labels in the module directories.
--spec generate([{ModuleName :: atom(), Labels :: list()}], #context{}) -> list().
+-spec generate([{ModuleName :: atom(), Labels :: list()}], #context{}) -> ok.
 generate(ModLabels, Context) ->
     generate1(ModLabels, Context).
 
+-spec generate1([{atom(), list()}], #context{}) -> ok.
```

Comment:
The spec in line 33 uses a record, please define a type for the record and use that instead.